fixed-price-subscriptions/client/react/package.json

```diff
@@ -36,8 +36,7 @@
   },
   "devDependencies": {
     "autoprefixer": "^9.8.0",
-    "postcss-cli": "^7.1.1",
-    "tailwindcss": "^1.4.6"
+    "postcss-cli": "^7.1.1"
   },
-  "proxy": "http://localhost:4242"
+  "proxy":"http://localhost:4242"
```

Comment:
Microscopic nit, but just flagging in case the removal of the space here wasn't intentional.